src/LiveStorage.node.test.ts

```diff
@@ -0,0 +1,10 @@
+import { LiveStorage } from './LiveStorage'
+
+test('REAME example', () => {
+  const scenario = () => {
+    // TODO: BroadcastChannel is not available in Node 14 nor `jsdom` environment
```

Comment:
Would you be interested if I push a work-in-progress state I've been working on, where we don't invoke things not available in NodeJS? You could then continue with converting a class into a singleton or discuss any other way to implement storage persistency in NodeJS.

Reply:
probably yes, not sure when I would find time to think about it properly...

Reply:
Absolutely no hurry in this. I've pushed my progress to the master: https://github.com/mswjs/live-storage/commit/994505decf95a392f0fd2bd40c252d54e8304620

Reply:
rebased and using the public property `value` to make the assertion for now

Reply:
IMHO `this.value` should be enough for storage persistence in Node for now,, enforcing singletons can be discussed in a separate issue whether needed (probably nice to have, but not sure if worth the complexity - if I am inspired by a simple idea to enforce single instance per id, I can write a separate PR later)